lib/ActivityStreams.js

```diff
@@ -443,6 +466,34 @@ ActivityStreams.prototype = {
   },
 
   /**
+    * Start a timer to fetch a new recommendation every hour
+    */
+  _startRecommendationRefresh(recommendationTTL) {
```

Comment:
if you only call it once, you can say you "start" something. If you call it repeatedly, why not just call it `_refreshRecommendation`?